setup.py

```diff
@@ -15,7 +15,7 @@
 
 
 # We use the version to construct the DOWNLOAD_URL.
-VERSION = '1.4.0'
+from krux_boto.version import VERSION
```

Comment:
Nit: I would prefer if the import statements are all at the top with corresponding comments like in `boto.py`.